.github/actions/run-snowplow-micro/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Run Snowplow micro
+runs:
+  using: "composite"
+  steps:
+    - name: Download Snowplow micro JAR
+      run: curl -L -o snowplow-micro.jar https://github.com/snowplow-incubator/snowplow-micro/releases/download/micro-1.2.1/snowplow-micro-1.2.1.jar
```

Comment:
More of curiosity, on CCI we use the .jar file as ENV Variable. Any reason for that ? If there's any, is needed here as well ?